src/test/run-pass-fulldeps/ttdelim-span.rs

```diff
@@ -0,0 +1,36 @@
+// Copyright 2015 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+extern crate syntax;
+
+use syntax::parse;
+use syntax::ast;
+use std::borrow::ToOwned;
+
+pub fn main() {
```

Comment:
Should this live in libsyntax's tests instead of run-pass? It looks like its specifically testing part of the syntax crate as opposed to the compiler as a whole.